packages/admin-next/dashboard/src/lib/client/promotions.ts

```diff
@@ -81,15 +81,22 @@ async function removePromotionRules(
   )
 }
 
-async function listPromotionRules(id: string | null, ruleType: string) {
+async function listPromotionRules(
+  id: string | null,
+  ruleType: string,
+  promotionType?: string
+) {
   return getRequest<PromotionRuleAttributesListRes>(
-    `/admin/promotions/${id}/${ruleType}`
+    `/admin/promotions/${id}/${ruleType}?promotion_type=${promotionType}`
```

Comment:
Should we do 
```suggestion
if(promotionType) {
const promoFilter = `promotion_type=${promotionType}`
...
```

Or does passing an empty `promotion_type=` work?